salt/modules/timezone.py

```diff
@@ -18,14 +18,22 @@
 
 log = logging.getLogger(__name__)
 
+__virtualname__ = 'timezone'
+
 
 def __virtual__():
     '''
     Only work on POSIX-like systems
     '''
     if salt.utils.is_windows():
-        return (False, 'The timezone execution module failed to load: not available on Windows systems.')
-    return True
+        return (False, 'The timezone execution module failed to load: '
+                       'not available on Windows systems.')
+
+    if salt.utils.is_darwin():
+        return (False, 'The timezone execution module failed to load: '
+                       'not available on MacOS systems.')
```

Comment:
If we have a mac_timezone module, this message should inform them of that fact instead of just telling them timezone functionality isn't available for their system.